test/jest/fixtures.js

```diff
@@ -63,6 +63,7 @@ module.exports = function registerFixtures(global) {
   });
 
   global.registerWorkerFixture('playwright', async({}, test) => {
+    Error.stackTraceLimit = 15;
```

Comment:
why is this needed? Seems like a strange place to put it.

Reply:
Our stacks are long and some tests match file names in them. We have it in test.js, I need it in the vm, hence the strange place. Will figure it out!